src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -441,38 +439,109 @@
-                         from-locked-amount]]
+         params [(merge {:uuid                 (str (random-uuid))
+                         :sendType             send-type
+                         :addrFrom             from-address
+                         :addrTo               to-address
+                         :amountIn             amount-in
+                         :amountOut            amount-out
+                         :tokenID              token-id
+                         :toTokenID            to-token-id
+                         :disabledFromChainIDs disabled-from-chain-ids
+                         :disabledToChainIDs   disabled-to-chain-ids
+                         :gasFeeMode           gas-rates
+                         :fromLockedAmount     from-locked-amount}
+                        rest-keys)]]
      {:db            (cond-> db
                        :always (assoc-in [:wallet :ui :send :amount] amount)
                        :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
                        :always (assoc-in [:wallet :ui :send :sender-network-values]
                                 sender-network-values)
                        :always (assoc-in [:wallet :ui :send :receiver-network-values]
                                 receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
                        :always (update-in [:wallet :ui :send] dissoc :network-links)
                        token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+      :json-rpc/call [{:method   "wallet_getSuggestedRoutesV2Async"
+                       :params   params
+                       :on-error (fn [error]
+                                   (rf/dispatch [:wallet/suggested-routes-error error])
+                                   (log/error "failed to get suggested routes (async)"
+                                              {:event  :wallet/get-suggested-routes
+                                               :error  error
+                                               :params params}))}]})))
+
+(rf/reg-event-fx :wallet/stop-get-suggested-routes
```

Comment:
Still haven't figured out where this would be useful

Reply:
Replaced clean calculation with this endpoint

https://github.com/status-im/status-mobile/pull/20631/files#diff-5ade62d5bf01b3e057aa185d7168680c84ad10ae0a01bf639e8fbe9e4804130cR11

Reply:
> Still haven't figured out where this would be useful

You used this event on unmount if I'm not mistaken. That seems good to me because it avoids needless computation of processing a signal that's not needed anymore.

Reply:
If we have `:wallet/stop-get-suggested-routes`, wdyt of refactoring the other event to `:wallet/start-get-suggested-routes` for consistency?